scripts/app.js

```diff
@@ -0,0 +1,356 @@
+const { promisify } = require("util");
+const exec = promisify(require("child_process").exec);
+const fs = require("fs");
+const dotenv = require("dotenv");
+const result = dotenv.config();
+
+if (result.error) {
+  if(result.error.errno === -2){
+    console.log("The .env file does not exist. You cannot stage a build without this file.")
+    return
```

Comment:
Technically this ran when I ran the script, but it's not typical to put a return outside of a function.  You could just use "process.exit()".